paper-api/src/main/java/io/papermc/paper/registry/data/InstrumentRegistryEntry.java

```diff
@@ -0,0 +1,89 @@
+package io.papermc.paper.registry.data;
+
+import io.papermc.paper.registry.RegistryBuilder;
+import io.papermc.paper.registry.TypedKey;
+import net.kyori.adventure.sound.Sound;
+import net.kyori.adventure.text.Component;
+import org.bukkit.MusicInstrument;
+import org.checkerframework.checker.index.qual.NonNegative;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Contract;
+
+/**
+ * A data-centric version-specific registry entry for the {@link org.bukkit.MusicInstrument} type.
+ */
+@ApiStatus.Experimental
+@ApiStatus.NonExtendable
+public interface InstrumentRegistryEntry {
+
+    @Contract(pure = true)
+    @NonNegative
```

Comment:
This is the wrong annotation, this should be positive and it can be inlined on the same line, same for range and the builder methods. Even if technically those annotations shouldn't even be used for floats.